Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -33,6 +33,8 @@ public class InteractableTests : BasePlayModeTests
         private const string DefaultInteractablePrefabAssetPath = "Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Prefabs/Model_PushButton.prefab";
         private const string RadialSetPrefabAssetPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/RadialSet.prefab";
         private const string PressableHoloLens2TogglePrefabPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButtonHoloLens2Toggle.prefab";
+        private const string RadialPrefabAssetPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/Radial.prefab";
+        private const string ToggleSwitchPrefabAssetPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ToggleSwitch.prefab";
```

Comment:
use?

Reply:
The `RadialPrefabAssetPath` is used in `AssembleRadialSet()`.  The `ToggleSwitchPrefabAssetPath` was used to test if the ToggleSwitch prefab worked with the new InteractableToggleCollection (it does work), but it was there for testing and is not used anywhere else, so it will be removed.

Reply:
Sorry I just meant ToggleSwitchPrefabAssetPath